crates/miden-lib/asm/kernels/transaction/lib/prologue.masm

```diff
@@ -1094,11 +1094,11 @@ end
 #!
 #! Where:
 #! - TX_SCRIPT_ROOT is the transaction's script root.
-#! - TX_SCRIPT_ARGS_KEY is the commitment of the transaction script args used to obtain them from 
+#! - TX_SCRIPT_ARGS_KEY is the commitment of the transaction script args used to obtain them from
 #!   the advice map.
 proc.process_tx_script_data
     # read the transaction script root from the advice stack
-    adv_loadw
+    padw adv_loadw
     # => [TX_SCRIPT_ROOT]
 
     # store the transaction script root in memory
```

Comment:
I think `dropw` on line 1105 below is not necessary (and actually having it there leads to overwriting top 4 stack elements).